Zend/zend_vm_def.h

```diff
@@ -1683,6 +1683,49 @@ ZEND_VM_HANDLER(136, ZEND_ECHO, CONST|TMPVAR|CV, ANY)
 	ZEND_VM_NEXT_OPCODE_CHECK_EXCEPTION();
 }
 
+// This is copied from ZEND_ECHO above with only a small part different
+ZEND_VM_HANDLER(200, ZEND_ECHO_ESCAPE, CONST|TMPVAR|CV, ANY)
+{
+	USE_OPLINE
+	zval *z;
```

Comment:
Removed a variable definition here that was removed in the function this was copied from